src/main/java/com/nirima/jenkins/plugins/docker/DockerContainerWatchdog.java

```diff
@@ -121,18 +137,23 @@ protected void execute(TaskListener listener) throws IOException, InterruptedExc
         Instant snapshotInstance = this.clock.instant();
         Map<String, Node> nodeMap = loadNodeMap();
         
-        for (Cloud c : getAllClouds()) {
-            if (!(c instanceof DockerCloud)) {
-                continue;
+        try {
+            for (Cloud c : getAllClouds()) {
```

Comment:
Isn't `getAllClouds()` returning a `List<DockerCloud>` these days?  As per earlier [review comment](https://github.com/jenkinsci/docker-plugin/pull/658/commits/3320a9a816cb8d1cf3f2003e46cea8da33a67b51#r192681087).

If so, we don't need to check that `c instanceof DockerCloud`, we can simply do `for (DockerCloud dc : getAllClouds()) {` instead.